s3cache_test.go

```diff
@@ -87,3 +87,29 @@ func TestCache(t *testing.T) {
 	_, err = cache.Get(ctx, "dummy")
 	assert.Equal(t, autocert.ErrCacheMiss, err)
 }
+
+func TestCacheWithPrefix(t *testing.T) {
+	testS3Cache := &testS3{cache: map[string][]byte{}}
+	cache := &Cache{bucket: "my-bucket", s3: testS3Cache}
+	cache.Prefix = "/path/to/certs/here/"
```

Comment:
This should be set in the initialization.

Reply:
I left this outside of the initialization because that is how it will be used. Once the Cache object is returned you add the Prefix, so I'm trying to mimic that.